src/apps/chifra/pkg/binary/binary.go

```diff
@@ -0,0 +1,24 @@
+package binary
+
+import (
+	"io"
+)
+
+type CacheUnmarshaler interface {
+	UnmarshalCache(version uint64, reader io.Reader) error
+}
+
+type CacheMarshaler interface {
+	MarshalCache(writer io.Writer) error
+}
+
+func reverseBytes(original []byte) (reversed []byte) {
```

Comment:
Can you add a comment why this is needed? Is this that GoLang issue you referenced elsewhere?

Reply:
This function is removed now